pmm-app/tests/pages/qanPage.js

```diff
@@ -52,6 +52,13 @@ module.exports = {
     tablesTabContents: "//div[@class='card-body']//pre",
     copyQueryButton: "//button[@id='copyQueryExample']",
     spinnerLocator: "//i[@class='fa fa-spinner fa-spin spinner']",
+    newQANPanelContent: '.panel-content',
+    countOfItems: '.css-ni562u',
+    resetAll: 'button#reset-all-filters',
+    newQANSpinnerLocator: "//i[@class='fa fa-spinner fa-spin spinner ant-spin-dot']",
+    showSelected: "//button[contains(@class, 'ant-btn css-x930nf ant-btn-link')]",
+    filterBy: "//span[contains(@class, 'css-1xgzgf8 ant-input-affix-wrapper')]//input",
+    showTop5OrAllLink: "//span[contains(@class, 'css-lljzu5') and contains(text(), '",
```

Comment:
We should not break a selector, so that one part can live here, and the other - somewhere else.
We could solve it by:
1/ writing a function like we did on L#345 or
2/ write a custom function using locate like we did [here](https://github.com/percona/grafana-dashboards/blob/PMM-2.0/pmm-app/tests/pages/pmmSettingsPage.js#L4)